### PR TITLE
feat: render new Homeview for artsy people

### PR DESCRIPTION
**Pull request by MounirDhahri**
This PR resolves [ONYX-1330] 

### Description

This PR exposes the new homeview to artsy people.

**For easier review, I suggest you hide white spaces**
 

### PR Checklist

- [x] I have tested my changes on **iOS** and **Android**.
- [x] I hid my changes behind a **[feature flag]**, or they don't need one.
- [x] I have included **screenshots** or **videos**, or I have not changed the UI.
- [x] I have added **tests**, or my changes don't require any.
- [x] I added an **[app state migration]**, or my changes do not require one.
- [x] I have documented any **follow-up work** that this PR will require, or it does not require any.
- [x] I have added a **changelog entry** below, or my changes do not require one.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to **run** this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

-

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

- render new Homeview for artsy people - mounir

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

[ONYX-1330]: https://artsyproduct.atlassian.net/browse/ONYX-1330?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/app/AppRegistry.tsx [hunk 1]
```diff
@@ -499,8 +498,7 @@ export const modules = defineModules({
       screenOptions: {
         statusBarTranslucent: true,
       },
-      fullBleed:
-        ArtsyNativeModule.isBetaOrDev && !unsafe_getFeatureFlag("ARPreferLegacyHomeScreen"),
+      fullBleed: true,
```
Comment (MounirDhahri) [inline src/app/AppRegistry.tsx:501]: I tested the screens after this change to make sure I didn't break anything

https://github.com/user-attachments/assets/8797283c-8a4b-4ada-b826-0dee8cd52053

File: src/app/Scenes/Home/HomeContainer.tsx [hunk 2]
```diff
@@ -15,8 +15,10 @@ export const InnerHomeContainer = () => {
   const showPlayground = useDevToggle("DTShowPlayground")
 
   const preferLegacyHomeScreen = useFeatureFlag("ARPreferLegacyHomeScreen")
+  const isArtsyEmployee = GlobalStore.useAppState((state) => state.auth.userHasArtsyEmail)
 
-  const shouldDisplayNewHomeView = ArtsyNativeModule.isBetaOrDev && !preferLegacyHomeScreen
+  const shouldDisplayNewHomeView =
+    (isArtsyEmployee || ArtsyNativeModule.isBetaOrDev) && !preferLegacyHomeScreen
```
Comment (MounirDhahri) [inline src/app/Scenes/Home/HomeContainer.tsx:21]: We want to keep the old logic + the condition of having an artsymail account account

**Comment by ArtsyOpenSource** [comment]
### This PR contains the following changes:

- Dev changes (render new Homeview for artsy people - mounir)
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against ac2f47eeb23d6ea23b9791453a6dde9d4b768366
</p>

File: src/app/AppRegistry.tsx (hunk 1 above)
Comment (dblandin) [inline src/app/AppRegistry.tsx:501]: Awesome 💯

File: src/app/Scenes/Home/HomeContainer.tsx (hunk 2 above)
Comment (dblandin) [reply src/app/Scenes/Home/HomeContainer.tsx:21]: Awesome 👍🏼

**Review by dblandin — APPROVED** [review]

File: src/app/Scenes/Home/HomeContainer.tsx (hunk 2 above)
Comment (anandaroop) [reply src/app/Scenes/Home/HomeContainer.tsx:21]: Wondering if this check is now out of sync with this one, which was added previously in #10808

https://github.com/artsy/eigen/blob/144858049f09e23b78fb6b68c922b450c1d742c2/src/app/utils/useHideSplashScreen.ts#L16-L18

Any cause for concern?

**Pull request merged**